<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMZ_TestWorkClass</class>
 <widget class="QMainWindow" name="MMZ_TestWorkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MMZ_TestWork</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="FibSequenceIndexLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Индекс последовательности:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="FibResultLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fib() = </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="FibSequenceSpinBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="FibTextBrowser">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>80</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MMZ_TestWork.qrc"/>
 </resources>
 <connections/>
</ui>
